applications/asset_tracker/README.rst

```diff
@@ -0,0 +1,6 @@
+.. _asset_tracker_template_redirect:
+
+Asset Tracker
+#############
+
+For new Asset Tracker projects, it is recommended to use the `Asset Tracker Template <Asset Tracker Template_>`_.
```

Comment:
This page is not really needed. It should be handled with a migration guide entry. The link can be mentioned there and in the changelog, and any access to this page covered by a redirect.
Also because we should not add fake README files for non-existent applications.

@divipillai , @umapraseeda , can we handle this redirect with TechDocs redirect?

Reply:
Discussed offline - The page is now removed and An entry is added in the migration guide. The Asset Tracker v2 ref link is retained in the migration guide. Uma will checking the possibility to link from Asset Tracker v2 doc page to the section in the migration guide with TechDocs redirect.